core/src/test/scala/org/apache/spark/sql/test/SharedSQLContext.scala

```diff
@@ -188,20 +210,20 @@ object SharedSQLContext extends Logging {
 
   private def initializeTiDB(forceNotLoad: Boolean = false): Unit =
     if (_tidbConnection == null) {
-      val jdbcUsername = getOrElse(_tidbConf, TiDB_USER, "root")
```

Comment:
the renaming seems not necessary. We actually use `jdbcUserName` to format a JDBC connection string.